### PR TITLE
Provides compatibility with Tachyon 0.8.2

### LINKED ISSUE #566: Notebook built with Spark 1.6.0 incompatible with Tachyon 0.8.2
The build with Spark 1.6.0 also contains the local instance of Tachyon 0.7.x which overrides the tachyon dependencies of 0.8.2.
Spark in order to work with tachyon must match exactly the same version.

In order to fix it I had to disable the execution of local tachyon and remove the 0.7.x libraries from the lib folder.

### PR DESCRIPTION
**Pull request by huitseeker**
Fixes #566

### CODE REVIEW

**Comment by huitseeker** [comment]
This is just a quick hack while I'm pressed for time : the code duplication induced by the tachyon_0.8.2 directory is dirty, and I'm open to suggestions on how to architect this better. Fundamentally, there's no need to duplicate any kind of code, this should rather be implemented using a version mask that includes 0.7.1 & 0.8.2.

File: modules/tachyon/src/main/tachyon_0.8.2 [hunk 1]
```diff
@@ -0,0 +1 @@
+tachyon_0.7.1
```
Comment (vidma) [inline modules/tachyon/src/main/tachyon_0.8.2]: why `tachyon_0.7.1` inside a file named `tachyon_0.8.2` ?

also is this `text-file` really intended to live inside `modules/src/main...` ?

**Comment by vidma** [comment]
looks :ok: (if it works)

**Comment by huitseeker** [comment]
To test: 
- download a stock, compiled spark 1.6, from the vanilla link http://d3kbcqa49mib13.cloudfront.net/spark-1.6.0-bin-hadoop2.6.tgz
- run it on your HDFS-equipped cluster, even with `enabledTachyon=false` (I tested mesos & yarn)
- launch a job from the spark-notebook built from this PR

File: modules/tachyon/src/main/tachyon_0.8.2 (hunk 1 above)
Comment (huitseeker) [inline modules/tachyon/src/main/tachyon_0.8.2]: Yeah. This might be an actual problem on Windows. I hate the duplicaiton creating the files would, though.

**Comment by jarutis** [comment]
Tested, the fix works :)

**Comment by vidma** [comment]
:ship:'ing then.

**Pull request merged**